applications/nrf_desktop/doc/led_stream.rst

```diff
@@ -0,0 +1,99 @@
++-----------------------------------------------+--------------------------------+                |                        |                                             |
+| :ref:`nrf_desktop_hids`                       | ``config_fetch_request_event`` |                |                        |                                             |
++-----------------------------------------------+                                |                |                        |                                             |
+| :ref:`nrf_desktop_usb_state`                  |                                |                |                        |                                             |
++-----------------------------------------------+--------------------------------+                +------------------------+---------------------------------------------+
+|                                               |                                |                | ``led_event``          | :ref:`nrf_desktop_leds`                     |
+|                                               |                                |                +------------------------+---------------------------------------------+
+|                                               |                                |                | ``config_fetch_event`` | :ref:`nrf_desktop_hids`                     |
+|                                               |                                |                |                        +---------------------------------------------+
+|                                               |                                |                |                        | :ref:`nrf_desktop_usb_state`                |
++-----------------------------------------------+--------------------------------+----------------+------------------------+---------------------------------------------+
+
+Configuration
+*************
+
+The module receives LED effects through the :ref:`nrf_desktop_config_channel` and displays them using the :ref:`nrf_desktop_leds`.
+For this reason, make sure that ``CONFIG_DESKTOP_LED_ENABLE`` and ``CONFIG_DESKTOP_CONFIG_CHANNEL_ENABLE`` are both set.
+
+To enable the module, use the ``CONFIG_DESKTOP_LED_STREAM_ENABLE`` Kconfig option.
+
+You can also define the stream LED event queue size (``CONFIG_DESKTOP_LED_STREAM_QUEUE_SIZE``).
+The queue is used by the module as data buffer for the data received from the host computer.
+
+Configuration channel
+*********************
+
+The module provides the following :ref:`nrf_desktop_config_channel` functionalities:
+
+* Displaying LED effects synchronized to the music played by the host computer.
+* Displaying random LED effects generated by the host computer.
+
+Use the following command to display led effects synchronized to the music from :file:`sample.wav`:
+
+.. code-block:: python
```

Comment:
I will change this to console, since this is a command line command (not a python code).